graphql_compiler/tests/test_data_tools/data_tool.py

```diff
@@ -16,3 +37,60 @@ def generate_data(client):
                     continue
 
                 client.command(sanitized_command)
+
+
+@retry(tries=20, timeout=1)  # pylint: disable=no-value-for-parameter
```

Comment:
`tries=20` seems very high, tests might take a long time to fail

Reply:
From local testing if I see about 15s to run the test from against cold start of the containers, so this gives us a little bit of overhead. Is 20s to failure too long? We give the OrientDB backend that long to startup as well.

Reply:
I guess it's okay if that's how long it takes. We might want to revisit in the future and see if we can make tests a bit faster -- a shorter feedback loop would speed up development.

Reply:
So just for reference here, I timed out 10 integration tests and once the fixture is loaded, they each take about 100 ms. The fixture loading is about 1s. So even with 50 integration tests, we're looking at < 10s all told.